store/src/java/com/zimbra/qa/unittest/TestDataSource.java

```diff
@@ -556,14 +579,15 @@ public void tearDown()
         cos.setDataSourcePollingInterval(mOriginalCosPollingInterval);
         cos.setDataSourcePop3PollingInterval(mOriginalCosPop3PollingInterval);
         cos.setDataSourceImapPollingInterval(mOriginalCosImapPollingInterval);
-
         cleanUp();
     }
 
     public void cleanUp()
     throws Exception {
         TestUtil.deleteAccount(TEST_USER_NAME);
-        TestUtil.deleteTestData(USER_NAME, NAME_PREFIX);
+        TestUtil.deleteAccount(USER_NAME_2);
+        //TestUtil.deleteTestData(USER_NAME, NAME_PREFIX);
```

Comment:
Does anyone know if this line is actually necessary?  With it in bad things happen but I wasn't sure if by removing it data was being left behind in the system.